recipes/amptk/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+cp -r ./* $PREFIX/
```

Comment:
We do not need all of these folders/files in the recipe, isn't it? Can you please explicitly copy the needed files?

Reply:
I see, I just did a local conda build and see that this currently "pollutes" the environment.  The `amptk` script is a wrapper for running several other scripts in the directories, so it does need to have the `lib`, `util`, `bin`, and `DB` folders.  But after seeing what `conda build` did locally that is not what we want to do.  Is it possible to keep the `amptk` files/package in subdirectory and then just softlink the `amptk` script into the `bin` folder?  That is how I install it on home-brew for example.